### ISSUE TITLE
An error always appears when editing a template

### ISSUE THREAD

**Issue by ghost** [czechboy0/Buildasaur#283]
Hi! For some reason Buildasaur started displaying an error message on a certain step of template editing. That step is always stuck in loading (the spinner in the corner doesn't disappear) and "Next" button is always disabled. See attached screenshot. Please let me know if any additional information is needed for diagnostics.

![screen shot 2016-06-21 at 17 50 49](https://cloud.githubusercontent.com/assets/13571000/16238536/b9b90242-37d8-11e6-8db3-924c337cf6db.png)

Buildasaur version 1.0.1, OS X 10.11.5 on client, OS X 10.11.4 on server, Xcode 7.3.1 on client, Xcode 7.3 on server. This error didn't appear before with the same combination of Xcode versions on client and server and looks like it broke after Buildasaur update.

**Comment by czechboy0** [comment]
Hi yes, I made changes so that invalid parsing doesn't crash Buildasaur anymore, now an error is shown. Unfortunately the description is not very descriptive, haha. Could you please take a look into the log at ~/Library/Application Support/Buildasaur? Hopefully that'll print the full text of the error. Thanks!

**Comment by czechboy0** [comment]
Also - note to self - make sure the error description is properly printed.

**Comment by ghost** [comment]
Hey, thanks for the quick reply. Unfortunately, I don't see anything related to the error in the logs. Last few lines are:

```
[VERBOSE]: Selected <redacted>
[VERBOSE]: Selected /Users/<redacted>
[INFO]: Sending heartbeat event ["event_type": heartbeat, "uptime": 20.01254892349243, "running_syncers": 0, "running_syncer_types": {
}]
[VERBOSE]: Finished blueprint validation with success!
[VERBOSE]: Selected Coverage
```

So the last line is about the scheme selection which was successful just before the error appeared.

**Comment by czechboy0** [comment]
Damn, okay. Could you try to clone the Buildasaur code and run it in Xcode? Otherwise I'm afraid there's no way to get to more details about the error.

**Comment by ghost** [comment]
Having a problem here trying to compile it:

```
% pod install   

 CocoaPods-Keys has detected a keys mismatch for your setup.
 What is the key for GitHubAPIClientId
 > 
```

What's the expected input for that?

**Comment by czechboy0** [comment]
Hmm just put anything in, if you don't use OAuth in Buildasaur (by e.g. providing your github token manually), you won't need this.

**Comment by ghost** [comment]
Hm, interesting, I didn't think it might be needed just at `pod install` stage. Will try what you said, thanks.

**Comment by czechboy0** [comment]
It's because I'm using CocoaPods Keys to store the private OAuth Buildasaur keys on my computer.

**Comment by MaxDesiatov** [comment]
I've created a PR that fixes this: https://github.com/czechboy0/XcodeServerSDK/pull/137

**Comment by valeriomazzeo** [comment]
I have the same issues, it worked fine before updating

**Comment by czechboy0** [comment]
Can either of you verify it's fixed in master of Buildasaur? I've updated XcodeServerSDK there

**Comment by valeriomazzeo** [comment]
I don't have any easy way to compile code on the server :S 
Also `pod install` yells:

> CocoaPods-Keys has detected a keys mismatch for your setup.
>  What is the key for GitHubAPIClientId

**Comment by voidrender** [comment]
@valeriomazzeo You can build and run Buildasaur on your workstation to test, or alternatively, you could build and then copy Buildasaur.app from the Products group to your CI server:
![buildasaur_xcworkspace](https://cloud.githubusercontent.com/assets/1653961/17633296/60fb2e60-6081-11e6-8158-5e7c5b33c26b.png)

As for the `pod install` warning, you'll need to give it answers for the [cocoapods-keys](https://github.com/orta/cocoapods-keys) entries. If you're using a GitHub personal access token to authenticate your syncer, you can enter anything you want.

**Comment by czechboy0** [comment]
@explicitcall Can you please verify this is fixed in 1.0.2? (Please close this issue if so, cheers)

**Comment by MaxDesiatov** [comment]
Fixed for me, thanks.

**Comment by MaxDesiatov** [comment]
I don't have a permission to close the issue though.

**Comment by valeriomazzeo** [comment]
I confirm this is fixed in 1.0.2
